src/styles/mainClassDefinitions.less

```diff
@@ -467,8 +461,8 @@
 	border-radius: 4px;
 	text-align: center;
 	margin-bottom: 10px;
-	margin-left: 5px;
 	margin-right: 5px;
+	width: 100%;
```

Comment:
Is this for the clip button? Wouldn't leave in the margin-left cause this to not take up all the space on the right?

Reply:
I'm not sure what you mean? The margin-left was causing this to pop out of alignment with the location label and the picker

Reply:
As discussed, we should be able to remove margin-right